modules/apps/feature-flag/feature-flag-web/src/FeatureFlags.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "feature flags"
```

Comment:
John can update this line and test.properties component.names list with `portal-feature-flags`